src/openforms/forms/tests/test_data_migrations.py

```diff
@@ -1,7 +1,13 @@
 from unittest import skip
 
+from django.apps import apps
+from django.db import IntegrityError
+from django.test import TestCase
+from django.utils.module_loading import import_string
+
 from openforms.formio.utils import iter_components
 from openforms.utils.tests.test_migrations import TestMigrations
+from ..models import Form, FormDefinition, FormStep, FormVariable
```

Comment:
isort isn't running properly, but would fail on this import

```suggestion
from openforms.formio.utils import iter_components
from openforms.utils.tests.test_migrations import TestMigrations

from ..models import Form, FormDefinition, FormStep, FormVariable
```